sgv2-docsapi/pom.xml

```diff
@@ -119,6 +119,11 @@
       <artifactId>jbool_expressions</artifactId>
       <version>1.23</version>
     </dependency>
+    <dependency>
```

Comment:
Just saw this, we are migrating to the open api v3 seagger annotations.. So it s not anymore `@ApiModelProperty`, but `@Schema` annotations. You can see example in the ExampleResource and its DTO model.. This dependency must go out.